pages/web-literacy/color.js

```diff
@@ -0,0 +1,33 @@
+var categories = require('./categories');
```

Comment:
Yeah, we probably don't need a colors object but I dunno if we want to remove it from circletree or not. Probably easier to always do it in css.